doc/source/myrocks/zenfs.rst

```diff
@@ -91,33 +91,47 @@ Configuration
    .. sourcecode:: bash
 
       lsblk -o NAME,SIZE,ZONED
-      NAME        SIZE  ZONED
-      sda       247.9G  none
-      |-sda1    230.9G  none
-      |-sda2        1G  none
-      |-sda3       16G  none
-      sdb        15.5T  host-managed
+      NAME                    SIZE  ZONED
+      sda                       247.9G  none
+      |-sda1                    230.9G  none
+      |-sda2                        1G  none
+      |-sda3                       16G  none
+      <short_block_device_name>   7.2T  host-managed
 
 		
-#. Change the ownership of ``nvme0n2`` to the ``mysql:mysql`` user account.
+#. Change the ownership of ``<block_device_name>`` to the ``mysql:mysql`` user account.
```

Comment:
remove the marks from <block_device_name> since this is a substitution.
There are several of these places where we should remove the marks.